### PR TITLE
TL translations for remote-branches.asc

### PR DESCRIPTION
**Pull request by josteinfrancois**
Completed the translations for remote-branches.asc. Made 1,118 words-worth translations for this contribution. Feel free to review my work. Thank you.

### CODE REVIEW

File: book/03-git-branching/sections/remote-branches.asc [hunk 1]
```diff
@@ -111,11 +111,9 @@ From https://github.com/schacon/simplegit
  * [new branch]      serverfix    -> origin/serverfix
 ----
 
-It's important to note that when you do a fetch that brings down new remote-tracking branches, you don't automatically have local, editable copies of them.
-In other words, in this case, you don't have a new `serverfix` branch -- you only have an `origin/serverfix` pointer that you can't modify.
+Importanteng tandaan na kagpag ikaw ay gumawa ng isang fetch na humihila pababa ng bagong remote-tracking na mga branch, hindi ka awtomatikong magkakaroon ng lokal, mababagong mga kopya nito. Sa ibang mga salita, sa kasong ito, wala kang isang bagong `serverfix` na branch -- mayroon ka lang isang `origin/serverfix` na pointer na hindi mo mababago.
```
Comment (heywens) [inline book/03-git-branching/sections/remote-branches.asc]: ``kagpag`` to ``kapag``

File: book/03-git-branching/sections/remote-branches.asc [hunk 2]
```diff
@@ -111,11 +111,9 @@ From https://github.com/schacon/simplegit
...
 
-To merge this work into your current working branch, you can run `git merge origin/serverfix`.
-If you want your own `serverfix` branch that you can work on, you can base it off your remote-tracking branch:
+Upnag ma-merge ang trabahong ito sa iyong kasalukuyang tinatrabahong branch, maaari mong patakbuhin ang `git merge origin/serverfix`. Kung gusto mong magkaroon ng sariling `serverfix` na branch kung saan ka pwedeng magtrabaho, maaari kang mag-base nito sa iyong remote-tracking na branch:
```
Comment (heywens) [inline book/03-git-branching/sections/remote-branches.asc]: ``Upnag`` to ``Upang``

**Review by heywens — COMMENTED** [review]
Hi @josteinfrancois, good translation. I only found 2 typos. Kindly check :)

File: book/03-git-branching/sections/remote-branches.asc (hunk 1 above)
Comment (josteinfrancois) [reply book/03-git-branching/sections/remote-branches.asc]: changed already

File: book/03-git-branching/sections/remote-branches.asc (hunk 2 above)
Comment (josteinfrancois) [reply book/03-git-branching/sections/remote-branches.asc]: done

**Comment by josteinfrancois** [comment]
Hey @heywens, thank you for pointing those out. It has been updated. Thank you! :)

**Pull request merged**